index.html

```diff
@@ -46,15 +46,16 @@
     <link rel="stylesheet" href="https://rsms.me/inter/inter.css" />
     <link href="/dist/output.css" rel="stylesheet" />
   </head>
-  <!-- overflow-y-hidden doesnt work with safari -->
+  <!-- overflow-y-hidden doesnt work with safari, only overflow-hidden -->
   <body
     x-data="{
       characterMenuOpen: false,
     }"
+    class="overflow-hidden flex flex-col"
```

Comment:
re-added overflow-hidden here, this was supposed to disable scroll on ios as well, but not wiggle